src/main/settings.ts

```diff
@@ -34,6 +47,7 @@ const DEFAULT_SETTINGS: UserSettings = {
   enableAutoUpdate: true,
   releaseChannel: "stable",
   selectedTemplateId: DEFAULT_TEMPLATE_ID,
+  workspaceZoomLevel: DEFAULT_WORKSPACE_ZOOM_LEVEL,
```

Comment:
remove this - you shouldn't need it (and if you change this you'll need to rebaseline a lot of e2e test snapshots)